OSDOCS-16949#CQA work Stor5 - CSI Vol Populators (#114642)
openshift/openshift-docs: modules/persistent-storage-csi-vol-populator.adoc

[id="persistent-storage-csi-vol-populator_{context}"]
= Volume populators overview

[role="_abstract"]

> Volume populators, with the `dataSourceRef` field, allow you to prepopulate volumes from a Custom Resource Definition (CRD) instead of only persistent volume claims (PVCs) and snapshots.

Reviewer: ```suggestion
With volume populators that use the `dataSourceRef` field, you can prepopulate volumes from a Custom Resource Definition (CRD) instead of only persistent volume claims (PVCs) and snapshots.
```
Avoid anthropomorphic language ("allow you"). Please check my revision to make sure that it is accurate.
